lte/protos/BUILD.bazel

```diff
@@ -354,6 +354,16 @@ cpp_grpc_library(
     deps = [":sms_orc8r_cpp_proto"],
 )
 
+proto_library(
```

Comment:
This comment is actually for line 372:
`diam_errors_cpp_proto` needs to be added as a dependency, i.e., after line 374 add
`deps = [":diam_errors_cpp_proto"]`

can be tested on vm or dev container via `bazel build lte/protos:subscriberauth_cpp_proto`. Or build all proto libraries via `bazel build lte/protos/...`.